native/monogame/directx12/CommandContext.cpp

```diff
@@ -0,0 +1,363 @@
+// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+#include "directx12.h"
+
+#include "CommandContext.h"
+#include "DeviceResources.h"
+#include "GraphicsEnums.h"
+#include "Texture.h"
+#include "Sampler.h"
+#include "Heaps.h"
+
+#if defined(_GAMING_XBOX_SCARLETT)
```

Comment:
So i decided after reviewing other open source projects it is fine for us to put Xbox code like this in public.

Microsoft does it in their open source repos all the time.  Other open source projects do as well.

https://github.com/search?q=_GAMING_XBOX&type=code